index.html

```diff
@@ -243,14 +243,15 @@ <h3 id="expression" data-export="" data-dfn-type="dfn">
 
 <section>
   <h3 id="verify" data-export="" data-dfn-type="dfn">
-The web must make it possible for people to verify the information they see
+The web must make it possible for people to verify, and assess the trustworthiness of, information.
```

Comment:
The commas here aren't quite right. I think we can just remove them?

```suggestion
The web must make it possible for people to verify and assess the trustworthiness of information.
```

Reply:
I think the meaning of the sentence is different with and without the commas (verifying information vs verifying trustworthiness of information). @hadleybeeman and I have proposed an alternative less clunky version below